Sources/SecureXPC/Server/XPCServiceServer.swift

```diff
@@ -40,9 +44,18 @@ internal class XPCServiceServer: XPCServer {
 
         return (uint32ToString(packageType.bigEndian), uint32ToString(packageCreator.bigEndian))
     }
-    
+
+    private lazy var xpcServiceName: String = {
+        // This is safe to unwrap because it was already checked in ``_forThisXPCService``.
+        Bundle.main.bundleIdentifier!
+    }()
```

Comment:
Is this indirection necessary? My understanding is was, quoting from Swift.org, "A lazy stored property is a property whose initial value isn’t calculated until the first time it’s used." If that's true then I think the following ought to be equivalent:
```swift
private lazy var xpcServiceName: String = Bundle.main.bundleIdentifier!
```

Reply:
Ever since the `guard` was removed from here, no, it's not necessary anymore.